crypto/fipsmodule/ec/ec_nistp.c

```diff
@@ -259,3 +259,44 @@ void ec_nistp_point_add(const ec_nistp_meth *ctx,
   cmovznz(z3, ctx->felem_num_limbs, z2nz, z1, z_out);
 }
 
+static int16_t get_bit(const EC_SCALAR *in, size_t in_bit_size, size_t i) {
```

Comment:
Nit: Sorry, I'm not well-versed in what this is actually doing, so `get_bit` might actually make sense here. But is it possible for this function to have a better name?

Reply:
added a comment spelling out what's the function doing:
```
// Returns i-th bit of the scalar (zero or one).
```